### PR TITLE
Check llvm version from the system #61

### PR DESCRIPTION
**Pull request by flipchan**
For issue #61 .

Installer scripts now checks if it can get the llvm version from the system, if it finds it, it will change it otherwise leave it as it is

### CODE REVIEW

File: ring/mk/llvm_check.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash  
+
+
+function llvm_version_check() {
```
Comment (h4x3rotab) [inline ring/mk/llvm_check.sh:4]: We require the version >= 10. Can we enforce that? Also, do you think we can also check the clang version?

**Review by h4x3rotab — COMMENTED** [review]
Thanks for the contribution! If merged, it will be eligible to our developer bounty (tips) on the blockchain. I left a few comments. Besides, can you also make sure to correctly set the CLANG and LLVM environment variable for [cc](https://crates.io/crates/cc) used in build.rs?

File: ring/mk/llvm_check.sh (hunk 1 above)
Comment (flipchan) [reply ring/mk/llvm_check.sh:4]: How about upgrading the version if the system is less then 10?

Comment (h4x3rotab) [reply ring/mk/llvm_check.sh:4]: I think it's fine to prompt user they have to upgrade. Automatic upgrade is a bit too aggressive I think

Comment (flipchan) [reply ring/mk/llvm_check.sh:4]: @h4x3rotab implemented a version  check in latest

**Comment by h4x3rotab** [comment]
Closed since nobody is following and nobody is reporting the problem.

**Pull request closed without merging**